tests/core/arch-arm64/stalker-arm64.c

```diff
@@ -346,6 +352,251 @@ store_x0 (GumCpuContext * cpu_context,
   *last_x0 = cpu_context->x[0];
 }
 
+STALKER_TESTCASE (exclude_bl)
+{
+  const guint32 code_template[] =
+  {
+    0xa9bf7bf3, /* push {x19, lr} */
+    0xd2801553, /* mov x19, #0xaa */
+    0xd2800883, /* mov x3, #0x44  */
+    0xd2800662, /* mov x2, #0x33  */
+    0xd2800441, /* mov x1, #0x22  */
+    0xd2800220, /* mov x0, #0x11  */
+    0xa9bf07e0, /* push {x0, x1}  */
+    0x94000009, /* bl func_a      */
+    0xa8c107e0, /* pop {x0, x1}   */
+    0xd2801103, /* mov x3, #0x88  */
+    0xd2800ee2, /* mov x2, #0x77  */
+    0xd2800cc1, /* mov x1, #0x66  */
+    0xd2800aa0, /* mov x0, #0x55  */
+    0x94000005, /* bl func_b      */
+    0xa8c17bf3, /* pop {x19, lr}  */
+    0xd65f03c0, /* ret            */
+
+    /* func_a: */
+    0xd2801100, /* mov x0, #0x88  */
+    0xd65f03c0, /* ret            */
+
+    /* func_b: */
+    0xd2801320, /* mov x0, #0x99  */
+    0xd65f03c0, /* ret            */
+  };
+
+  StalkerTestFunc func;
+  guint8 * func_a_address;
+
+  fixture->sink->mask = GUM_EXEC;
+  g_assert_cmpuint (fixture->sink->events->len, ==, 0);
```

Comment:
Let's move these two lines down to right before `follow_and_invoke()`, to make the logic at the end clearer.